src/WP_CLI/ParseThemeNameInput.php

```diff
@@ -158,6 +158,17 @@ protected function is_active_theme( $theme ) {
 		return $theme->get_stylesheet_directory() === get_stylesheet_directory();
 	}
 
+	/**
+	 * Check whether a given theme is the active theme parent.
+	 *
+	 * @param string $theme Theme to check.
+	 *
+	 * @return bool Whether the provided theme is the active theme.
+	 */
+	protected function is_active_parent_theme( $theme ) {
+		return $theme->get_stylesheet_directory() === get_template_directory();
```

Comment:
I don't think this is correct. This would mean "the parent IS the child theme". That doesn't seem to be what you want to check for here.

Reply:
It is a little confusing but it is correct.

The check: is the `$theme` (moina in the test case) the _parent_ of the currently active theme (as defined by `get_template_directory()`).

Reply:
Ah, I misread the code. I thought it used the `get_stylesheet_directory()` function, and not the `WP_Theme::get_stylesheet_directory()` method... NVM.